jme3-core/src/main/java/com/jme3/system/AppSettings.java

```diff
@@ -1350,4 +1351,65 @@ public boolean isUseRetinaFrameBuffer() {
     public void setUseRetinaFrameBuffer(boolean useRetinaFrameBuffer) {
         putBoolean("UseRetinaFrameBuffer", useRetinaFrameBuffer);
     }
+    
+    /**
+     * Get the Center Window state
+     *
+     * @return true for center the window in the middle of the screen
+     * @see #setCenterWindow(boolean)
+     */
+    public boolean getCenterWindow() {
+        return getBoolean("CenterWindow");
+    }
+    
+    /**
+     * True to enable center the window.
+     *
+     * @param center whether to center the window or not.
+     */
+    public void setCenterWindow(boolean center) {
+        putBoolean("CenterWindow", center);
+    }
+
+    
+    /**
+     * Get the position of the window's X Position
+     *
+     * @return int screen coordinates of the X Position of the window
+     * @see #setWindowXPosition(int)
+     */
+    public int getWindowXPosition() {
+        return getInteger("windowXpos");
```

Comment:
For consistency with the other settings, I think "windowXpos" should be capitalized and spelled out: "WindowXPosition".

Reply:
What do you think bob0bob?

Reply:
I changed it.

Reply:
It's still starting with a lower-case "w" somehow.